src/Components/Generator/CompanyWidget.tsx

```diff
@@ -0,0 +1,117 @@
+                <Accordion title={company['name']} id="company-info" expanded={true}>
+                    <div className="company-info-content">
+                        <div className="company-info-params">
+                            {company['address'] ? (
+                                [
+                                    <br />,
+                                    <span className="company-info-label">
+                                        <Text id="address-colon" />
+                                        &nbsp;
+                                    </span>,
+                                    <span>{company['address'].split('\n').join(', ')}</span>,
+                                ]
+                            ) : (
+                                <></>
+                            )}
+                            {company['fax'] ? (
+                                [
+                                    <br />,
+                                    <span className="company-info-label">
+                                        <Text id="fax-colon" />
+                                        &nbsp;
+                                    </span>,
+                                    company['fax'],
+                                ]
+                            ) : (
+                                <></>
+                            )}
+                            {company['email'] ? (
+                                [
+                                    <br />,
+                                    <span className="company-info-label">
+                                        <Text id="email-colon" />
+                                        &nbsp;
+                                    </span>,
+                                    company['email'],
+                                ]
+                            ) : (
+                                <></>
+                            )}
+                            {company['pgp-fingerprint'] ? (
+                                [
+                                    <br />,
+                                    <span className="company-info-label">
+                                        <Text id="pgp-fingerprint-colon" />
+                                        &nbsp;
+                                    </span>,
+                                    <code>
+                                        {company['pgp-url'] ? (
+                                            <a href={company['pgp-url']}>{company['pgp-fingerprint']}</a>
+                                        ) : (
+                                            company['pgp-fingerprint']
+                                        )}
+                                    </code>,
+                                ]
+                            ) : (
+                                <></>
+                            )}
+                        </div>
+                        {comments && comments.length > 0 ? (
```

Comment:
As TS correctly points out, this might lead to bugs, if `comments` is `undefined`, because `undefined > 0` is not well typed: https://stackoverflow.com/a/71340806